cmd/process-agent/collector.go

```diff
@@ -81,18 +85,27 @@ func NewCollector(cfg *config.AgentConfig) (Collector, error) {
 	}
 
 	enabledChecks := make([]checks.Check, 0)
+	runRealTime := !ddconfig.Datadog.GetBool("process_config.disable_realtime_checks")
 	for _, c := range checks.All {
+		if !runRealTime && isRealTimeCheck(c.Name()) {
+			log.Infof("Skip enabling check '%s': realtime disabled", c.Name())
+			continue
+		}
 		if cfg.CheckIsEnabled(c.Name()) {
 			c.Init(cfg, sysInfo)
 			enabledChecks = append(enabledChecks, c)
 		}
 	}
 
-	return NewCollectorWithChecks(cfg, enabledChecks), nil
+	return NewCollectorWithChecks(cfg, enabledChecks, runRealTime), nil
+}
+
+func isRealTimeCheck(checkName string) bool {
+	return checkName == config.RTContainerCheckName
```

Comment:
Don't we need to add `RTProcessCheckName` here as well?

Reply:
I can add it, but it's the same `Process` check.

Reply:
Gotcha. Thanks for clarifying it!